dev/create_release_doc.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+#
+# Copyright 2018-2023 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+from github import Github
+import rich_click as click
+
+class DocRelease:
+
+    def __init__(self, repo, tag: str):
+        self.repo = repo
+        self.tag: str = tag
+        self.release: str = ''
+        self.release_date: str = ''
+
+    def get_release(self):
+        self.release = self.repo.get_release(self.tag).body
+        self.release_date = self.repo.get_release(self.tag).created_at.strftime('%Y-%m-%d')
+
+    def create_doc(self):
+        doc = f'# {self.tag} - {self.release_date}' + '\n' * 2 + self.release
+        print(doc)
+
+@click.command()
+@click.option(
+    '--token', type=str, default=''
+)
+@click.option(
+    '--tag', type=str, default=''
+)
+
+def main(
```

Comment:
Maybe just add some doc that the output can be used to document the release.

Reply:
Added it. Thank you.